src/components/BoardHeader/BoardHeader.scss

```diff
@@ -90,15 +85,18 @@
     transform: scale(1);
   }
 }
+.board-header__share-button > svg {
+  height: 36px;
+  width: 36px;
```

Comment:
I guess 32px is too small? (in case it's not `$icon--large: 32px;`)

Reply:
I think it's too small. Looks strange in contrast to the larger avatars